build.sh

```
@@ -1,6 +1,14 @@
 #!/bin/sh
 echo "Setup webui files.."
-sh setup-webui.sh
+current_wd=$(pwd)
+static_dir="$(dirname $0)/src/main/resources/static"
+
+rm -rf ${static_dir} || echo ""
+cd ${current_wd}/webui
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)